import-export-cli/docs/apictl_get_policies_api.md

```diff
@@ -13,22 +13,21 @@ apictl get policies api [flags]
 ### Examples
 
 ```
-apictl get policies api -e production
-apictl get policies api -e prod --all
-apictl get policies api -e prod -l 10
-apictl get policies api -e prod --format jsonArray
-NOTE: The flag (--environment (-e)) is mandatory
-NOTE: Flags (--all) and (--limit (-l)) cannot be used at the same time
+apictl get policies api -e dev
+apictl get policies api -e dev --all
+apictl get policies api -e dev -l 30
+ NOTE: The flag (--environment (-e)) is mandatory
+ NOTE: Flags (--all) and (--limit (-l)) cannot be used at the same time
 ```
 
 ### Options
 
 ```
-  -e, --environment string   Environment of the API Policies to be fetched
-      --format string        Pretty-print API Policies using Go Templates. Use "jsonArray" to list all fields
-  -h, --help                 help for API Policies
-  -l, --limit string         Limit the number of policies fetched
-  --all                      Fetch all available API Policies
+      --all                  Get all API Policies
```

Comment:
Formatting issue? unnecessary leading white spaces?

Reply:
No, `--all` does not have the shorthand value (i.e  flag with only `-` not `--`). Same as this one https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/docs/apictl_export.md?plain=1#L35